src/app/user-page/user-page.component.ts

```diff
@@ -25,14 +27,20 @@ export class UserPageComponent extends Base implements OnInit {
   public googleProfile: Profile;
   public gitHubProfile: Profile;
   public loggedInUserIsAdminOrCurator: boolean;
+  protected otherLinkedAccountsInfo: AccountInfo[] = [];
+  accountsInfo: Array<AccountInfo> = Object.assign([], accountInfo);
```

Comment:
This creates a new array, but it doesn't create copies of the array elements, so the the code on line 91 is still modifying the global `AccountInfo` objects.